### PR TITLE
CNFT2-1416 Integrate reorder API

### PR DESCRIPTION
**Pull request by jlee-enquizit**
Here we integrate the reorder API to commit reordering changes to the db

### CODE REVIEW

File: apps/modernization-ui/src/apps/page-builder/context/DragDropProvider.tsx [hunk 1]
```diff
@@ -301,9 +312,9 @@ const DragDropProvider: React.FC<{ children: React.ReactNode; data: PagesTab | u
         } else if (type === 'question') {
             handleQuestionMove(source, destination);
         } else {
-            // else its a Subsection move so we go here
             handleSubsectionMove(source, destination);
         }
+        reorderObjects(token, Number(destination.droppableId) + (destination.index - 1), moveId, pageDropId);
```
Comment (mpeels) [inline apps/modernization-ui/src/apps/page-builder/context/DragDropProvider.tsx:317]: Question (blocking): Passing `Number(destination.droppableId) + (destination.index - 1)` as the `after` component Id seems wrong. I haven't had my hands on the reorder UI side, but it seems that we would want to get the new index of the drop location.

If thats > 0, get the Id of component at new index - 1.
If drop index == 0, then we want the parent container Id.

If I'm misunderstanding something I may need you to give me a walkthrough of whats going on. I can be easily confused.

File: apps/modernization-ui/src/apps/page-builder/pages/EditPage/EditPage.tsx [hunk 2]
```diff
@@ -31,6 +31,7 @@ export const EditPage = () => {
     useEffect(() => {
         if (pageId) {
             fetchPageDetails(token, Number(pageId)).then((data) => {
+                console.log('DATA', data);
```
Comment (mpeels) [inline apps/modernization-ui/src/apps/page-builder/pages/EditPage/EditPage.tsx:34]: Issue: console.log

**Review by mpeels — CHANGES_REQUESTED** [review]

**Review by mpeels — CHANGES_REQUESTED** [review]
Reordering of Section does not seem to work. 
*Note - I did merge main into the branch locally when testing


reorderBug

**Comment by jlee-enquizit** [comment]
> Reordering of Section does not seem to work. *Note - I did merge main into the branch locally when testing
> 
> reorderBug [ reorderBug ](https://user-images.githubusercontent.com/109251240/281786254-40cb7b1a-b158-48f8-875f-67d9d94c69b3.gif) [ ](https://user-images.githubusercontent.com/109251240/281786254-40cb7b1a-b158-48f8-875f-67d9d94c69b3.gif)

Fixed!

**Review by mpeels — APPROVED** [review]

**Pull request merged**